tests/playwright/pages/editor-page.js

```diff
@@ -130,4 +130,119 @@ module.exports = class EditorPage extends BasePage {
+		await this.page.selectOption( '.elementor-control-_mask_size >> select', 'custom' );
+		await this.page.locator( '.elementor-control-_mask_size_scale .elementor-control-input-wrapper input' ).fill( '30' );
+		await this.page.selectOption( '.elementor-control-_mask_position >> select', 'top right' );
+	}
+
+	/**
+	 * Autopopulate the Image Carousel.
+	 *
+	 * @return {Promise<void>}
+	 */
+	async populateImageCarousel() {
+		await this.page.locator( '[aria-label="Add Images"]' ).click();
+
+		// Open Media Library
+		await this.page.click( 'text=Media Library' );
+
+		// Upload the images to WP media library
+		await this.page.setInputFiles( 'input[type="file"]', './tests/playwright/resources/A.jpg' );
+		await this.page.setInputFiles( 'input[type="file"]', './tests/playwright/resources/B.jpg' );
+		await this.page.setInputFiles( 'input[type="file"]', './tests/playwright/resources/C.jpg' );
+		await this.page.setInputFiles( 'input[type="file"]', './tests/playwright/resources/D.jpg' );
+		await this.page.setInputFiles( 'input[type="file"]', './tests/playwright/resources/E.jpg' );
+
+		// Create a new gallery
+		await this.page.locator( 'text=Create a new gallery' ).click();
+
+		// Insert gallery
+		await this.page.locator( 'text=Insert gallery' ).click();
+
+		// Open The Additional options Section
+		await this.page.click( '#elementor-controls >> :nth-match(div:has-text("Additional Options"), 3)' );
+
+		// Disable AutoPlay
+		await this.page.selectOption( 'select', 'no' );
+	}
+
+	/**
+	 * Set a background color to an element.
+	 *
+	 * @param {string} color - The background color code;
+	 * @param {string} elementType - The element types are `widget` and `container`;
+	 *
+	 * @return {Promise<void>}
+	 */
+	async setBackgroundColor( color = '#A81830', elementType = 'widget' ) {
+		const panelTab = 'container' === elementType ? 'style' : 'advanced',
+			backgroundSelector = 'container' === elementType ? '.elementor-control-background_background ' : '.elementor-control-_background_background ',
+			backgroundColorSelector = 'container' === elementType ? '.elementor-control-background_color ' : '.elementor-control-_background_color ';
+
+		await this.activatePanelTab( panelTab );
+
+		if ( 'widget' === elementType ) {
+			await this.page.locator( '.elementor-control-_section_background .elementor-panel-heading-title' ).click();
+		}
+
+		await this.page.locator( backgroundSelector + '.eicon-paint-brush' ).click();
+		await this.page.locator( backgroundColorSelector + '.pcr-button' ).click();
+		await this.page.locator( '.pcr-app.visible .pcr-interaction input.pcr-result' ).fill( color );
+	}
+
+	/**
+	 * Remove the focus from the test elements by creating two new elements.
+	 *
+	 * @param {string} elementType - The element types are `section` and `container`;
+	 *
+	 * @return {Promise<void>}
+	 */
+	async removeFocus( elementType = 'section' ) {
+		await this.addElement( { elType: elementType }, 'document' );
```

Comment:
Can you trigger a click somewhere outside of the element instead?

Reply:
@udidol updated